packages/playwright-ct-react17/hooks.d.ts

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
-export declare function beforeMount<HooksConfig>(
+export interface RegisterHooksConfig {}
```

Comment:
Note: I have not changed the `react-router.spec.tsx` for React17 to show that declaration merging is completely optional